eternalcore-core/src/main/java/com/eternalcode/core/feature/vanish/VanishService.java

```diff
@@ -34,4 +36,13 @@ public boolean isVanished(Player player) {
         }
         return false;
     }
+
+    @FeatureDocs(
+        name = "Vanish tabulation",
```

Comment:
Trochę unrelated jesteśmy w serwisie od vanisha a nie tabulacji. Permisja jest w sumie też myląca jak ktoś se doda vanish.see to nadal nie będzie mógł widzieć graczy